<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskForm</class>
 <widget class="QWidget" name="TaskForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>11</y>
     <width>359</width>
     <height>243</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="typeLabel">
      <property name="text">
       <string>Type</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="comboBox">
      <item>
       <property name="text">
        <string>Event</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Task</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="nameLabel">
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="name"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="locationLabel">
      <property name="text">
       <string>Location</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="location"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="DescritionLabel">
      <property name="text">
       <string>Description</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QTextEdit" name="description">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="beginLabel">
      <property name="text">
       <string>Begin</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QDateTimeEdit" name="beginDateTime">
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="expireLabel">
      <property name="text">
       <string>Expire</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QDateTimeEdit" name="expireDateTime">
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="repeatLabel">
      <property name="text">
       <string>Repeats</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="repeats"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
